### PR TITLE
SIG member roles: TOC Liaison role description and responsibilities

### PR DESCRIPTION
**Pull request by alena1108**
cc @lizrice

### CODE REVIEW

**Comment by quinton-hoole** [comment]
LGTM!  Thanks Alena.

**Comment by caniszczyk** [comment]
LGTM thanks

File: sigs/cncf-sigs.md [hunk 1]
```diff
@@ -114,6 +114,14 @@ As a starting point let’s be inspired by CNCF OSS Projects and by K8s SIGs.  T
 * Leads projects in the SIG’s area.
 * Has the time and ability to perform deep technical dives on projects.   Projects may include formal CNCF projects or other projects in the area covered by the SIG.
 
+
+#### TOC Liaison
+
+* Streamlines communications between TOC and SIG chairs.
+* Communicates performance of the SIG to Governing Board & TOC.
```
Comment (lizrice) [inline sigs/cncf-sigs.md:121]: Maybe just to TOC, since GB don't have direct oversight of the SIG

File: sigs/cncf-sigs.md [hunk 2]
```diff
@@ -114,6 +114,14 @@ As a starting point let’s be inspired by CNCF OSS Projects and by K8s SIGs.  T
 * Leads projects in the SIG’s area.
 * Has the time and ability to perform deep technical dives on projects.   Projects may include formal CNCF projects or other projects in the area covered by the SIG.
 
+
+#### TOC Liaison
+
+* Streamlines communications between TOC and SIG chairs.
```
Comment (lizrice) [inline sigs/cncf-sigs.md:120]: As part of this, maybe we should add something about helping the SIG set priorities

Comment (alena1108) [reply sigs/cncf-sigs.md:120]: @lizrice will submit suggested changes in the subsequent pr

**Comment by quinton-hoole** [comment]
Very good idea.

**Pull request merged**